src/domains/wallet/components/ReceiveFunds/__snapshots__/ReceiveFunds.test.tsx.snap

```diff
@@ -257,12 +256,7 @@ exports[`ReceiveFunds should render without a wallet name 1`] = `
           </div>
           <div
             class="mt-8"
-          >
-            <img
```

Comment:
QRCode disappeared here. Shouldn't it still be present in `ReceiveFunds should render without a wallet name`?

Reply:
Seems strange indeed. Let me double check on Receivefunds.test and fix it

Reply:
Async rendering was not properly catched in ReceiveFunds tests. Fixed now. Thanks @faustbrian!